viral/learning_stages.py

```
@@ -1,133 +1,904 @@
+                    & (df["rewarded"] == rewarded)
+                ]["landmark_tuning"],
+            ).pvalue
+            p_text = f"P = {p_value:.2g}"
+            axes[idx].text(i, text_y, p_text, ha="center", va="top")
+
+    plt.tight_layout(rect=[0, 0, 1, 0.95])
+    plt.savefig(
+        SERVER_PATH
+        / "viral_plots"
+        / "landmark_tuning"
+        / f"landmark_tuning_comparison_plot.png"
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)